composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/config": "^2.8 || ^3.2 || ^4.0",
         "symfony/console": "^2.8 || ^3.2 || ^4.0",
         "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
-        "symfony/event-dispatcher": "^2.8 || ^3.2 || ^4.0",
```

Comment:
Wouldn’t this drop support for SF 2 and 3 ?

Reply:
Refs #592 592